R/digest.R

```diff
@@ -57,8 +57,8 @@ digest <- function(object, algo=c("md5", "sha1", "crc32", "sha256", "sha512",
     is_streaming_algo <- algo == "spookyhash"
 
     if (is_streaming_algo && !serialize) {
-        .errorhandler(paste0(algo, " algorithm is not available without serialization."),  # #nocov
-                      mode=errormode)                                                      # #nocov
+        return(.errorhandler(paste0(algo, " algorithm is not available without serialization."),  # #nocov
```

Comment:
this seemed to just be missing - not sure if there was a specific reason why this one should *not* be a return?

Reply:
Good catch.  Had to check the file -- that is indeed what is done in a few other invocations. 

:+1: